pkg/pillar/cmd/domainmgr/metric.go

```diff
@@ -17,9 +17,10 @@ func metricsTimerTask(ctx *domainContext, hyper hypervisor.Hypervisor) {
 	log.Functionln("starting metrics timer task")
 	getAndPublishMetrics(ctx, hyper)
 
-	// Publish 4X more often than zedagent publishes to controller
+	// Publish 20X more often than zedagent publishes to controller
```

Comment:
should here be an minimum beyond which we do not reduce the metric interval? Just in case user sets `timer.metric.interval` to 1 second. But that's a topic for a different PR

Reply:
There is (hopefully) a min enforced for the setting in global.go